components/cluster/command/deploy.go

```diff
@@ -55,30 +43,9 @@ var (
 	errDeployNameDuplicate = errNSDeploy.NewType("name_dup", errutil.ErrTraitPreCheck)
 )
 
-type (
-	componentInfo struct {
-		component string
-		version   string
-	}
-
-	deployOptions struct {
-		user              string // username to login to the SSH server
-		identityFile      string // path to the private key file
-		usePassword       bool   // use password instead of identity file for ssh connection
-		ignoreConfigCheck bool   // ignore config check result
-	}
-
-	hostInfo struct {
-		ssh  int    // ssh port of host
-		os   string // operating system
-		arch string // cpu architecture
-		// vendor string
-	}
-)
-
 func newDeploy() *cobra.Command {
-	opt := deployOptions{
-		identityFile: path.Join(tiuputils.UserHome(), ".ssh", "id_rsa"),
+	opt := deploy.DeployOptions{
```

Comment:
Prefer to rename the `DeployOptions` to `Options` according to Golang idiomatic.

Reply:
Ignore this comment.